### PR TITLE
Feature/k8s

### PR DESCRIPTION
**Pull request by feikesteenbergen**
For now, more a request for comments

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -83,15 +84,15 @@ ENV WALE_ENV_DIR=$PGHOME/etc/wal-e.d/env
...
+ADD scm-source.json configure_spilo.py launch.sh postgres_backup.sh patroni_wait.sh callback_role.py /
 ADD supervisor.d /etc/supervisor/conf.d/
 ADD stunnel.d /etc/stunnel
 ADD pgq_ticker.ini $PGHOME
 ADD motd /etc/
 RUN echo "source /etc/motd" >> /root/.bashrc
 RUN chmod 700 /postgres_*
 
-ENV ETCD_DISCOVERY_DOMAIN=postgres.acid.example.com DEBUG=0 LC_ALL=en_US.utf-8
+ENV ETCD_DISCOVERY_DOMAIN=postgres.acid.example.com DEBUG=1 LC_ALL=en_US.utf-8
```
Comment (feikesteenbergen) [inline postgres-appliance/Dockerfile]: Do we want to debug by default?

I'd be ok with that.

Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: DEBUG=1?

File: postgres-appliance/Dockerfile [hunk 2]
```diff
@@ -52,17 +52,18 @@ ENV PATH=$PATH:/usr/lib/postgresql/${PGVERSION}/bin
...
             daemontools lzop \
             # Required for /usr/local/bin/patroni
-            python python-pkg-resources \
+            python3 python3-pkg-resources \
             ${BUILD_PACKAGES} \
-    && pip install pip --upgrade \
-    && pip install --upgrade requests wal-e pystache patroni==$PATRONIVERSION \
+    && pip3 install pip --upgrade \
+    && pip3 install --upgrade requests pystache patroni==$PATRONIVERSION \
+    && pip3 install --upgrade gcloud boto wal-e==1.0.0a1 \
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile]: Should we introduce something like WAL_E_VERSION on the line 53 and use it here, like PATRONIVERSION?

**Comment by alexeyklyukin** [comment]
👍

File: postgres-appliance/configure_spilo.py [hunk 3]
```diff
@@ -106,7 +113,9 @@ def deep_update(a, b):
       parameters:
         archive_mode: "on"
         archive_timeout: 1800s
+        {{#USE_WALE}}
```
Comment (feikesteenbergen) [inline postgres-appliance/configure_spilo.py]: If we have:
```yaml
archive_mode: "on"
archive_command: 
```

Then xlog will accumulate. We can also set 
```yaml
archive_mode: "off"
```
if we don't have an actual archive_command.

Comment (alexeyklyukin) [inline postgres-appliance/configure_spilo.py]: Good point, thank you. Fixed.

**Comment by feikesteenbergen** [comment]
👍

**Comment by feikesteenbergen** [comment]
👍

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**